lib/discourse.rb

```diff
@@ -204,6 +204,22 @@ def self.unofficial_plugins
     plugins.find_all { |p| !p.metadata.official? }
   end
 
+  def self.find_plugins(args)
+    plugins.find_all do |plugin|
```

Comment:
I tend to prefer `select` over `find_all` because it's shorter and I don't confuse it with `all?`

```suggestion
    plugins.select do |plugin|
```